index.html

```diff
@@ -5,18 +5,19 @@
 	<title>Brafé 2</title>
 	<meta charset="utf-8">
 	<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
-	<link rel="stylesheet" type="text/css" href="/css/style.css">
+	<link rel="stylesheet" type="text/css" href="/Puro/css/style.css">
```

Comment:
Recomendo que se for usar capitalize nas pastas , coloque como padrao no projeto

Reply:
Pode deixar!